src/show/pages/public/Explore/index.js

```diff
@@ -0,0 +1,89 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { Button, ProgressBar } from '../../../components';
+
+const Explore = () => {
+  const [projectCard, setProjectCard] = useState([]);
+  const [selectedCategory, setSelectedCategory] = useState('');
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    axios
+      .get(`http://localhost:3000/v1/projects?category=${selectedCategory}`)
+      .then(res => setProjectCard(res.data))
+      .catch(err => err);
+  }, [selectedCategory]);
+
+  const categories = [
+    'garden',
+    'house',
+    'tech',
+    'cars',
+    'office',
+    'audio',
+    'backpacks',
+    'clothes'
+  ];
```

Comment:
Create a new folder in process called constants and import it from there.

Reply:
Also sort them abc cuz why not.

Reply:
done